op-e2e/celo/src/chain.js

```diff
@@ -1,8 +1,8 @@
-import { chainConfig } from 'viem/op-stack'
-import { defineChain } from 'viem'
+import { chainConfig } from "viem/op-stack";
```

Comment:
Why the switch from single to double quotes here? The other js files in this dir seem to all be using single quotes

Reply:
You're right - this was mainly due to my editor doing automatic formats on save.
I now added a `.prettierrc.toml` config and formatted all files consistently.

Reply:
Looks like the tab width of 4 is not working out here, there are now about 7k lines of changed code, I think 2 is probably the right value, or whatever doesn't reformat any of the codebase.

Reply:
Also, let's move out the re-formatting to a separate PR if needed.